src/screens/ConnectScreen.js

```diff
@@ -1,17 +1,25 @@
 import React from 'react';
 import Icon from '@expo/vector-icons/FontAwesome';
-import { StyleSheet, Text, View } from 'react-native';
+import { StyleSheet, Text, View, TouchableHighlight } from 'react-native';
 import { imageStyles } from '../styles/images';
 import { layoutStyles } from '../styles/layout';
 
 export default class ConnectScreen extends React.Component {
 
+  navigateToSignUp() {
+    console.log("connect screen");
+    console.log(this.props.navigation.state);
+    this.props.navigation.navigate('ConnectSignUp', this.props.navigation.state.params);
+  }
+
   render() {
     return (
       <View style={layoutStyles.flexCenter}>
         <Text>This is the Connect screen!</Text>
+        <TouchableHighlight onPress={ () => { this.navigateToSignUp(); } }>
```

Comment:
if the user is already on_connect, then we should not navigate to sign up. you should add a check for that before navigating to sign up. lmk if you are already doing this and i missed it.